lesson26/src/js/login.js

```diff
@@ -1,12 +1,55 @@
 import * as validation from "./validation";
 import { togglePassword } from "./module/toggle-password";
+import { Chance } from "chance";
+const chance = new Chance();
 
 const submitButton  = document.querySelector('[data-id="submit_button"]');
+const userNameInput = document.getElementById('username');
+const passwordInput = document.getElementById('current-password');
+const togglePasswordButton = document.getElementById('js-toggle-password');
 
 submitButton.addEventListener("click", (e) => {
     e.preventDefault();
+    login();
 });
 
+const isRegisteredUser = (inputValues) => {
+    const userData = {
+        name:"Nakagawa",
+        email: "nakagawa@sample.com",
+        password: "N302aoe3"
+    }
+
+    return ( inputValues.userName === userData.name || userData.email ) && inputValues.password === userData.password;
+}
```

Comment:
Returns a boolean value indicating whether the input value and the fixed value are the same or not.